src/import/prepare.rs

```diff
@@ -177,19 +214,16 @@ impl<Client: ImportClient> PrepareRangeJob<Client> {
                 // that PD cannot create scatter operator for the new split
                 // region because it doesn't have the meta data of the new split
                 // region.
-                for i in 0..SCATTER_WAIT_MAX_RETRY_TIMES {
-                    if self.client.has_region_id(new_region.region.id)? {
-                        if i > 0 {
-                            debug!("waited between split and scatter"; "retry times" => %i);
-                        }
-                        break;
-                    } else if i == SCATTER_WAIT_MAX_RETRY_TIMES - 1 {
-                        warn!("split region still failed after exhausting all retries");
-                    } else {
-                        thread::sleep(Duration::from_millis(SCATTER_WAIT_INTERVAL_MILLIS));
-                    }
-                }
+                exec_with_retry!(
+                    "split",
+                    self.client.has_region_id(new_region.region.id)?,
+                    SPLIT_WAIT_MAX_RETRY_TIMES,
```

Comment:
seems that no need to make them configurable, maybe we can just wrap them in macro.